### ISSUE TITLE
Unit test failures: billesholm.osm file is missing and test files written on project root folder

### ISSUE THREAD

**Issue by deniger** [neo4j-contrib/spatial#357]
I'd like to propose a PR to migrate geotools from 14.4 to 21.0. See: 

- https://github.com/deniger/spatial
- https://github.com/deniger/spatial/commit/0e1c67a17d6003cfd65b92815d4a06b96c25252d

Before creating this PR, I launched unit tests and I have an issue with org.neo4j.gis.spatial.TestSpatial:
```java.io.FileNotFoundException: target\osm\billesholm.osm```

**I believe that the file billesholm.osm is missing. Where could I find it ?**

Another points:
it seems that some files generated by unit tests are stored in project root  folder:
![image](https://user-images.githubusercontent.com/47569605/55629452-885bf080-57b3-11e9-9f24-7e28851d78e4.png)
( I can try to modify this point and let unit test generate files in the target folder).

Last point: unit tests are  not launched by travis. Should I modifiy travis to launch them ?

**Comment by craigtaverner** [comment]
I like all suggestions.

* Messy tests that output stuff outside target should be cleaned up to output to target
* It would be good if travis CI ran unit tests, then mistakes like the missing billesholm.osm would have been caught
* Fantastic that you've done work to port to newer geotools, and I'm looking forward to the PR to review.

**Comment by deniger** [comment]
I ll push my PR on Monday ;).
Could I ignore the test using the file billeshom or is there any chance to retrieve the file?

Then I could propose a new PR for unit test and Travis.

Happy to help :)



---- On Sat, 06 Apr 2019 12:35:59 +0200 notifications@github.com wrote ----



I like all suggestions.

Messy tests that output stuff outside target should be cleaned up to output to target
It would be good if travis CI ran unit tests, then mistakes like the missing billesholm.osm would have been caught
Fantastic that you've done work to port to newer geotools, and I'm looking forward to the PR to review.

**Comment by craigtaverner** [comment]
Yes, ignore the test. I will find the file and unignore the test when I do.

**Comment by craigtaverner** [comment]
Also, for me to merge your PR's, please make sure to agree to the CLA, as described at https://neo4j.com/developer/cla/

**Comment by deniger** [comment]
done




---- On Sun, 07 Apr 2019 00:49:18 +0200 Craig Taverner <notifications@github.com> wrote ----



Also, for me to merge your PR's, please make sure to agree to the CLA, as described at https://neo4j.com/developer/cla/
